scripts/abi.js

```diff
@@ -1,8 +1,6 @@
 const semver = require('semver')
 
 const nodeAbiRelation = {
-  12: '72',
-  13: '79',
   14: '83',
   15: '88',
   16: '93',
```

Comment:
should be included 20 and 21 here?

Reply:
If this was working without it... why do we have this `abi.js`? is this script used? 

Maybe we were using it before prebuildify?